packages/core/README.md

```diff
@@ -1,340 +1,67 @@
-@StartupActivity
-public class MainActivity extends ReactActivity {
-    ...
-}
-```
-
-##### Optional:
-
-This is a simplified process for React Native projects. If you want further information on the Android SDK integration, you can check the steps on the Native [Android Quick Integration Guide](https://docs.embrace.io/docs/android-integration-guide).
-
-### Step 3: Add the embrace-io npm package to your React Native Project
-
-In order to access the native functionalities that the Embrace SDKs provide, we built a wrapper using an npm package, fully compatible with react-native linking logic.
-Open your terminal and on the root of your React Native project install the npm package by running the following:
-
-```console
-$ npm i -S embrace-io
-```
-
-### Step 4: Link embrace-io
-
-React Native offers a pretty handy command that automatically links native modules with you React Native application.
-For more info, you can get it on the [Original Documentation Site](https://facebook.github.io/react-native/docs/linking-libraries-ios).
-
-To link embrace-io, open your terminal and on your React Native project root run:
-
-```console
-$ react-native link embrace-io
-```
-
-### Troubleshoot Step - Deprecated instruction on you app's build gradle file.
-
-During the linking process, React native will add a line to your android `app/build.gradle` file like this:
-
-```gradle
-...
-compile project(':embrace-io')
-```
-
-Be aware that the compile instruction has been deprecated but React Native still uses it on the Linking process.
-If that line is causing issues, you can replace it with:
-
-```gradle
-...
-implementation project(':embrace-io')
-```
-
-`implementation` is the right instruction for the compiler to add dependency to the project.
-You can read more about this [here](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation)
-
-### Step 5: Install latest pods
-
-Once you're done linking, it's necessary to download the pods that were linked to the project.
-To do so, go into the iOS project folder inside of your React Native app directory, where the Podfile should be located, and run:
-
-```console
-$ pod install
-```
-
-This should download `RNEmbrace` pod to your iOS Project.
-
-### Step 6: Call Methods
-
-A good place to start is after your startup moment ends. This moment starts automatically with the code added to AppDelegate.m on iOS or MainActivity.java on Android, and you should end on Component mount similar to the below code.
+Initialize method applies the necessary listener to your application. This allow Embrace to track javascript errors, check js bundle changes (if you use OTA), track js patch and react native versions.
 
 ```javascript
-import { endAppStartup } from "embrace-io";
+import { initialize } from "@embrace-io/react-native";
 
 type Props = {};
 export default class App extends Component<Props> {
   componentDidMount() {
-    endAppStartup();
+    initialize();
   }
 }
 ```
```

Comment:
You maybe can use the hooks version in the docs

Reply:
updated